integrated_channels/cornerstone/utils.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""
+Utilities for Cornerstone integrated channels.
+"""
+
+from __future__ import absolute_import, unicode_literals, with_statement
+
+from logging import getLogger
+
+from integrated_channels.cornerstone.models import CornerstoneLearnerDataTransmissionAudit
+
+LOGGER = getLogger(__name__)
+
+
+def create_cornerstone_learner_data_transmission_audit(request):  # pylint: disable=invalid-name
+    """
+        updates or creates CornerstoneLearnerDataTransmissionAudit
+    """
+    try:
+        course_id = request.GET['courseId']
```

Comment:
`course_id` is not available in the querystring. I think you should use the value if `course_key` variable returned by `RouterView.get_path_variables(**kwargs)`